scripts/prerm.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Stop systemd service
-if [ -d /run/systemd/system ] && [ "$1" = remove ]; then
+if [ -d /run/systemd/system ] && ( [ "$1" = remove ] || [ "$1" = "0" ] ); then
```

Comment:
Same as above with the parentheses.

As for the quotes I don't think it matters. But to keep it the same as the rest you can add quotes.